caikit/runtime/__init__.py

```diff
@@ -11,3 +11,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# Local
+from .http_server import RuntimeHTTPServer
```

Comment:
I think we actually _explicitly_ want to not do this. Having this import proactively will cause all of the HTTP server dependencies to become required for _any_ usage of `caikit.runtime` (including `caikit.runtime.grpc_server`)

Reply:
ah that makes sense